src/main/java/teammates/common/datatransfer/CourseRoster.java

```diff
@@ -5,8 +5,11 @@
 import java.util.List;
 import java.util.Map;
 
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
```

Comment:
Previously, I thought we can do it without involving the question attribute and response attributes.

How about we do this.

For response side, we leave it there.

For comment side, let's do it the right way - use roster to get the emailName/emailTeamname mapping.